packages/amplify-graphql-migration-tests/src/test-case-registry.ts

```diff
@@ -0,0 +1,46 @@
+import { TestEntry } from './test-case-types';
+
+/**
+ * REGISTER TEST CASES HERE
+ */
+export const getTestCaseRegistry = (): TestEntry[] => [
+  {
+    name: 'basic bi-di has-one and has-many connection',
+    schema: basicBiDiRelationSchema,
+  },
+  /*
```

Comment:
Should this be commented out?

Reply:
yeah this is meant to be a comment showing how to add more schemas to the test registry